include/arc_utilities/ros_helpers.hpp

```diff
@@ -82,6 +82,23 @@ namespace ROSHelpers
         return param_val;
     }
 
+    template <typename T>
+    inline Maybe::Maybe<T> GetParamOptional(const ros::NodeHandle& nh, const std::string& param_name, const std::string& calling_fn_name)
+    {
+        ROS_DEBUG_STREAM_NAMED("params", "No default value for " << param_name << ": Value must be on paramter sever");
```

Comment:
This message seems out of place.
Someone may want to call `GetParamOptional` regardless of default values

Reply:
Agreed.